README.md

```diff
@@ -1,3 +1,13 @@
+# ⚠️ **This repository is outdated!** ⚠️
+
+---
+
+As of version 4.0.0, this project has been merged into the ml toolkit mono-repo. For the latest updates and active development, please visit [https://github.com/KxSystems/ml](https://github.com/KxSystems/ml). 
```

Comment:
```suggestion
Since 9th October 2024 (version 4.0.0), this project has been merged into the ml toolkit mono-repo. For the latest updates and active development, please visit [https://github.com/KxSystems/ml](https://github.com/KxSystems/ml). 
```